src/website/src/app/documentation/demos/header/header-types.demo.html

```diff
@@ -125,7 +125,7 @@
                 <clr-dropdown>
                     <button class="nav-text" clrDropdownTrigger>
                         john.doe@vmware.com
-                        <clr-icon shape="caret down"></clr-icon>
+                        <clr-icon shape="caret down" aria-label="open menu"></clr-icon>
```

Comment:
This is an interesting one, as the content doesn't indicate what the button does. How does it read this in the screen reader? Again, the label probably belongs on the button instead of this icon.